app/lib/widgets/CustomAvatar.dart

```diff
@@ -35,61 +35,53 @@ class CustomAvatar extends StatelessWidget {
               color: AppCommonTheme.primaryColor,
             ),
           );
-        } else {
-          if (snapshot.hasData && snapshot.requireData.isNotEmpty) {
-            return CircleAvatar(
-              backgroundImage: MemoryImage(
-                Uint8List.fromList(snapshot.requireData),
-              ),
-              radius: radius,
-            );
-          } else {
-            if (isGroup) {
-              return CircleAvatar(
-                radius: radius,
-                backgroundColor: Colors.grey[700],
-                child: SvgPicture.asset('assets/images/people.svg'),
+        }
+        if (snapshot.hasData && snapshot.requireData.isNotEmpty) {
+          return CircleAvatar(
+            backgroundImage: MemoryImage(
```

Comment:
This should be replaced with ```CachedMemoryImage```.

Reply:
Fixed it, but it seems to be not cached.
You can run app to confirm it.

Reply:
Yeah caching doesn't seems to be working properly with ```CachedMemoryImage``` too. In 23 API env this bug is not there, only occurs in 32+ above APIs.  We have to dig deeper into this.

Reply:
No, I tested on api 23 and found caching is not working.

Reply:
Strange, I didn't found issues. Anyways, the bug is confirmed and also causes native crashes. I will open ticket for this.

Reply:
My side had no native crash.